Makefile

```diff
@@ -68,6 +71,22 @@ $(GO_BIN)/commander:
 $(GO_BIN)/auth0:
 	@$(MAKE) install
 
+#-----------------------------------------------------------------------------------------------------------------------
+# Assets
+#-----------------------------------------------------------------------------------------------------------------------
+.PHONY: assets
+
+assets: ## Generate Universal Login embeddable assets
+	${call print, "Generating Universal Login embeddable assets"}
+	@if [ ! -d "${UNIVERSAL_LOGIN_ASSETS_EXTERNAL_DIR}" ]; \
```

Comment:
Maybe use `mkdir -p` to create the directory if it doesn't exist already?

Reply:
Creating the directory is not enough, we need to actually clone the web app repo as well, which is not public, hence why we only check for the existence.